python/orca/src/bigdl/orca/data/shard.py

```diff
@@ -463,6 +463,45 @@ def zip(self, other):
                               "The two SparkXShards should have the same number of elements "
                               "in each partition")
 
+    def to_sparkdf(self):
```

Comment:
need to check if this is an XShards of Pandas Dataframe; otherwise report error

Reply:
Fixed